fdbrpc/include/fdbrpc/LoadBalance.actor.h

```diff
@@ -311,6 +389,7 @@ struct RequestData : NonCopyable {
 			modelHolder = Reference<ModelHolder>(new ModelHolder(model, stream->getEndpoint().token.first()));
 			response = stream->tryGetReply(request);
 			maybeDuplicateTSSRequest(stream, request, model, response, alternatives, channel);
+			maybeDoReplicaComparison(stream, request, model, response, alternatives, channel);
```

Comment:
Since this function can be called for retried request, i.e., `secondRequestData.startRequest()`, maybe we should avoid duplicating the comparison here?

Reply:
> Since this function can be called for retried request

Correct. To avoid duplication, maybe we should wait till "loadBalance()" receives a reply and then invoke the comparison actor only on the received reply?

Reply:
Which means that the comparison actor would be invoked in "loadBalance()" instead (only on the received reply)?

Reply:
Or, the other option is to cancel this actor on the request which hasn't replied yet once a reply is received from the other request? But that might still cause the duplication of work in the worst case, so maybe the suggestion above is a better choice?